uv.lock

```diff
@@ -42,7 +42,8 @@ dev = [
     { name = "dill" },
     { name = "docker" },
     { name = "docutils" },
-    { name = "moto", extra = ["awslambda", "batch", "cloudformation", "dynamodb", "s3", "server"] },
+    { name = "moto", version = "5.0.28", source = { registry = "https://pypi.org/simple" }, extra = ["awslambda", "batch", "cloudformation", "dynamodb", "s3", "server"], marker = "python_full_version < '3.9'" },
+    { name = "moto", version = "5.1.0", source = { registry = "https://pypi.org/simple" }, extra = ["awslambda", "batch", "cloudformation", "dynamodb", "s3", "server"], marker = "python_full_version >= '3.9'" },
```

Comment:
I saw this happen locally, why did it lock to two versions now?

Reply:
Because moto 5.1.0 does not support Python 3.8. The lock file takes that into account and locks the most appropriate version for each environment.

Reply:
wow, so smart, I'm really loving uv ;)